samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
I'm not even sure why we need to have this.  So looking at the internal usages of Ignore today, its the OIDC.SkipUnrecognizedRequests(which otherwise would Fail).  Why can't we just return Skip instead here?  

This seems totally not worth the complexity it brings, if we want to keep this as a quirk in OIDC events, fine, but I don't think this needs to spread.